deploy/docker/README.md

```diff
@@ -0,0 +1,18 @@
+######################
+
+### pump.io http://pump.io Docker image - listens port 443
+
+1) Copy pump.io.json.sample to pump.io.json and configure pump.io.json according to the needs
```

Comment:
Ultimately we want to publish a reusable image to Docker Hub so I don't think this is the way to go. We need to make stuff configurable at runtime. (Unless what I'm thinking of isn't actually possible? 95% sure it is though)